mne/export/tests/test_export.py

```diff
@@ -140,12 +133,28 @@ def test_double_export_edf(tmp_path):
         "dbs",
         "bio",
     ]
-    info = create_info(len(ch_types), sfreq=1000, ch_types=ch_types)
-    info = info.set_meas_date("2023-09-04 14:53:09.000")
-    data = rng.random(size=(len(ch_types), 1000)) * 1e-5
+    if stim_channel_index is not None:
+        ch_types.insert(stim_channel_index, "stim")
+    ch_names = np.arange(len(ch_types)).astype(str).tolist()
+    info = create_info(ch_names, sfreq=1000, ch_types=ch_types)
+    data = rng.random(size=(len(ch_names), 2000)) * 1e-5
+    return RawArray(data, info)
+
+
+def _pytest_mark_skipif_edfio_not_installed():
+    return pytest.mark.skipif(
+        not _check_edfio_installed(strict=False), reason="edfio not installed"
+    )
```

Comment:
A simpler pattern is just to do:
```
edfio_mark = pytest.mark.skipif(...)
```
then each function can just be decorated with
```
@edfio_mark
def test...
```